subprojects/testng/testng.gradle

```diff
@@ -1,5 +1,6 @@
 apply plugin: 'java'
 apply plugin: 'maven'
+apply from: "$rootDir/gradle/license.gradle"
```

Comment:
Coudn't we apply it automatically to all projects that apply Java plugin? Similar way it is done with JaCoCo - to do not miss another module once added.

Reply:
Uh, yeah maybe. Not a Gradle expert, but I can give it a shot tomorrow!

Reply:
Done. Please take a look :+1: